config/locales/en.yml

```diff
@@ -125,4 +125,8 @@ en:
         notice: "Achievement %{title} was updated."
       destroy:
         notice: "Achievement %{title} was deleted."
-
+  notification_mailer:
+    notification:
+      greeting: "Hello"
+      regards: "Best Regards"
+      team_name: "The Coursemology Team"
```

Comment:
Pls name these translation keys properly, I think here should be <code>signature</code> other than <code>team_name</code>.

Same with the <code>regards</code> line, maybe <code>leave-taking</code>?